core/src/main/java/io/aiven/klaw/helpers/db/rdbms/SelectDataJdbc.java

```diff
@@ -1881,9 +1881,13 @@ public boolean existsSchemaRequest(
       String requestOperationType,
       String env,
       int tenantId) {
-    return schemaRequestRepo
-        .existsByTenantIdAndEnvironmentAndRequestStatusAndRequestOperationTypeAndTopicname(
-            tenantId, env, requestStatus, requestOperationType, topicName);
+
+    boolean retval =
+        schemaRequestRepo
+            .existsByTenantIdAndEnvironmentAndRequestStatusAndRequestOperationTypeAndTopicname(
+                tenantId, env, requestStatus, requestOperationType, topicName);
+    log.info("the boolean answer {}", retval);
```

Comment:
do we want to log this statement

Reply:
No this was added while i was debugging that create/promote issue will remove now as it is not needed